test-utils/src/lib.rs

```diff
@@ -75,6 +79,38 @@ pub async unsafe fn get_hash_set<T, R: RpcConnection>(rpc: &mut R, pubkey: Pubke
     HashSet::from_bytes_copy(&mut account.data[8 + mem::size_of::<T>()..]).unwrap()
 }
 
+/// Fetches the fiven account, then copies and serializes it as a
+/// `ConcurrentMerkleTree`.
+pub async fn get_concurrent_merkle_tree<T, R, H, const HEIGHT: usize>(
+    rpc: &mut R,
+    pubkey: Pubkey,
+) -> ConcurrentMerkleTreeCopy<H, HEIGHT>
+where
+    R: RpcConnection,
+    H: Hasher,
+{
+    let account = rpc.get_account(pubkey).await.unwrap().unwrap();
+
+    ConcurrentMerkleTreeCopy::from_bytes_copy(&account.data[8 + mem::size_of::<T>()..]).unwrap()
+}
```

Comment:
it feels like we re-implement these wrappers in multiple places is this true?

Reply:
Only in two places:

* here (`test-utils/src/lib.rs`) which is meant just for tests. It's for convenience, because it fetches the account and serializes it in one run. I wish it was possible in the on-chain code. :smiling_face_with_tear: 
* On-chain in `account-compression/src/state.rs`. Which are meant for the on-chain usage.
  * https://github.com/Lightprotocol/light-protocol/blob/vadorovsky/mt-remove-arrays/programs/account-compression/src/state/address.rs#L71-L108
  * https://github.com/Lightprotocol/light-protocol/blob/vadorovsky/mt-remove-arrays/programs/account-compression/src/state/public_state_merkle_tree.rs

But I get your point - I implemented `*_copy_*` wrappers also in the on-chain code and were are not used. I removed them now. Now the copy wrapper is only here and zero-copy wrappers are on-chain.